src/WinRT.Runtime/Projections/PropertyChangedEventArgs.cs

```diff
@@ -1,175 +1,178 @@
-        }
-
-        public static IntPtr FromManaged(global::System.ComponentModel.PropertyChangedEventArgs value)
-        {
-            if (value is null)
-            {
-                return IntPtr.Zero;
-            }
-            return CreateMarshaler2(value).Detach();
-        }
-
-        public static void DisposeMarshaler(IObjectReference m) { m?.Dispose(); }
-        public static void DisposeAbi(IntPtr abi) { MarshalInspectable<object>.DisposeAbi(abi); }
-
-        public static unsafe MarshalInterfaceHelper<global::System.ComponentModel.PropertyChangedEventArgs>.MarshalerArray CreateMarshalerArray(global::System.ComponentModel.PropertyChangedEventArgs[] array) => MarshalInterfaceHelper<global::System.ComponentModel.PropertyChangedEventArgs>.CreateMarshalerArray2(array, CreateMarshaler2);
-        public static (int length, IntPtr data) GetAbiArray(object box) => MarshalInterfaceHelper<global::System.ComponentModel.PropertyChangedEventArgs>.GetAbiArray(box);
-        public static unsafe global::System.ComponentModel.PropertyChangedEventArgs[] FromAbiArray(object box) => MarshalInterfaceHelper<global::System.ComponentModel.PropertyChangedEventArgs>.FromAbiArray(box, FromAbi);
-        public static void CopyAbiArray(global::System.ComponentModel.PropertyChangedEventArgs[] array, object box) => MarshalInterfaceHelper<global::System.ComponentModel.PropertyChangedEventArgs>.CopyAbiArray(array, box, FromAbi);
-        public static (int length, IntPtr data) FromManagedArray(global::System.ComponentModel.PropertyChangedEventArgs[] array) => MarshalInterfaceHelper<global::System.ComponentModel.PropertyChangedEventArgs>.FromManagedArray(array, FromManaged);
-        public static void DisposeMarshalerArray(MarshalInterfaceHelper<global::System.ComponentModel.PropertyChangedEventArgs>.MarshalerArray array) => MarshalInterfaceHelper<global::System.ComponentModel.PropertyChangedEventArgs>.DisposeMarshalerArray(array);
-        public static unsafe void DisposeAbiArray(object box) => MarshalInspectable<object>.DisposeAbiArray(box);
-
-        public static string GetGuidSignature()
-        {
-            return "rc(Microsoft.UI.Xaml.Data.NotifyPropertyChangedEventArgs;{4f33a9a0-5cf4-47a4-b16f-d7faaf17457e})";
-        }
-    }
-}
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+using System;
+using System.ComponentModel;
+using System.Runtime.InteropServices;
+using WinRT;
+using WinRT.Interop;
+
+namespace ABI.Microsoft.UI.Xaml.Data
+{
+    internal sealed unsafe class PropertyChangedEventArgsRuntimeClassFactory
+    {
+        private readonly ObjectReference<IUnknownVftbl> _obj;
+
+        public PropertyChangedEventArgsRuntimeClassFactory()
+        {
+            if (FeatureSwitches.UseWindowsUIXamlProjections)
+            {
+                _obj = ActivationFactory.Get("Windows.UI.Xaml.Data.PropertyChangedEventArgs").As<IUnknownVftbl>(IID.IID_WUX_PropertyChangedEventArgsRuntimeClassFactory);
```

Comment:
nit: this can be addressed after, but we probably want to avoid the additional `As` call and object construction by doing what was done here: https://github.com/microsoft/CsWinRT/blob/c3c4aea1aced3d25e6f142096e599e3000f0740a/src/WinRT.Runtime/Projections/NotifyCollectionChangedEventArgs.cs#L27-L33

Reply:
Good catch, fixed in 000ca9bef419e0fdb3e90df1696e8b0ca341043a 👍